app/views/events/edit.html.erb

```diff
@@ -11,3 +13,17 @@
     <% end %>
   </div>
 </div>
+
+
+
+<script>
+  document.addEventListener("DOMContentLoaded", function() {
+    const triggerButton = document.querySelector(".trigger-copy");
+    const realCopyButton = document.getElementById("real-copy-button");
+    
+    triggerButton.addEventListener("click", function(e) {
+      e.preventDefault(); 
+      realCopyButton.click(); 
+    });
+  });
```

Comment:
let's keep using single quotes for JS

Reply:
fixed in ba5c424814bf34e0bfcfb12bef7cde9c423e8544